dat/autonav.lua

```diff
@@ -633,8 +659,9 @@ end
 -- Going for the landing approach
 function autonav_spob_land_brake ()
    local ret = ai.brake()
+
    if player.tryLand(false)=="impossible" then
-      return autonav_abort()
+      return autonav_abort("cannot land")
```

Comment:
```suggestion
      return autonav_abort(_("cannot land"))
```
Should be translated, as this is used in a format string which is translated before formatted.